jest/front/jest.setup.js

```diff
@@ -1,11 +1,15 @@
 require('@testing-library/jest-dom/extend-expect');
 const { matchers } = require('@emotion/jest');
 
+jest.mock('next/router');
```

Comment:
you're not using `next-router-mock` in this POC ?

Reply:
https://github.com/jpb06/mui-rhf-sandbox/pull/18/commits/db6792be8966c6f4f3348ea9e5b49ae4c0d3849c